### PR TITLE
Add Open Uri Initializer

### PR DESCRIPTION
**Pull request by jelaniwoods**
According to the new additions to the [project checklist](https://firstdraft.gitbook.io/teacher-s-manual/updating-old-projects-checklist), this branch adds:
* open_uri initializer similar to [this one](https://github.com/appdev-projects/omnicalc-actions/blob/master/config/initializers/open_uri.rb)

### CODE REVIEW

File: template.rb [hunk 1]
```diff
@@ -241,6 +241,11 @@ def render_file(filename)
     end
   end
 
+  initializer 'open_uri.rb', <<-CODE
```
Comment (LoganDSPrice) [inline template.rb]: I don't have familiarity with templates. Would it be easy to describe what this is doing? I have a few questions:
1. I see reference to `open_uri.rb` but I don't see that file anywhere in this repo. Is this line saying create this file in the `initializers` folder and that file contains the one line of `require "open-uri"`?
1. What is this syntax with ` <<-CODE` followed by another `CODE`? Is it pushing code into `open-uri.rb` with the `<<`?
1. Does that mean that `CODE` is being used to define a block?

Comment (jelaniwoods) [reply template.rb]: 1. I followed the Rails Guide on [application templates](https://edgeguides.rubyonrails.org/rails_application_templates.html#vendor-lib-file-initializer-filename-data-nil-block). This creates a new file in the `initializers/` folder. I had issues creating the file the other ways. I'll try again once master is up to speed.

1. This is heredoc syntax. I wrote a [blog post](https://medium.com/@jelaniwoods/what-is-a-heredoc-7a9515ad9848) about it. 

1.  Kinda

Comment (LoganDSPrice) [reply template.rb]: Awesome! Thanks!

**Comment by LoganDSPrice** [comment]
LGTM! :shipit:

**Pull request merged**